boom! testing cpython-json-0.1.1 against stable for beta
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/beta/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/beta/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! 8d4514314df7aa0324acbaff96e570654ec39997185c6022b029e56948c0fa27
boom! running `docker start -a 8d4514314df7aa0324acbaff96e570654ec39997185c6022b029e56948c0fa27`
kablam!    Compiling cpython v0.1.0
kablam!    Compiling python3-sys v0.1.3
blam! Build failed, waiting for other jobs to finish...
kablam! error: failed to run custom build command for `python3-sys v0.1.3`
kablam! process didn't exit successfully: `/target/debug/build/python3-sys-93e61270195fe162/build-script-build` (exit code: 101)
kablam! --- stderr
kablam! thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "failed to run python interpreter `\"python\" \"-c\" \"import sys; import sysconfig; print(sys.version_info[0:2]); print(sysconfig.get_config_var(\\\'LIBDIR\\\')); print(sysconfig.get_config_var(\\\'Py_ENABLE_SHARED\\\')); print(sysconfig.get_config_var(\\\'LDVERSION\\\')); print(sys.exec_prefix);\"`: No such file or directory (os error 2)"', /buildslave/rust-buildbot/slave/stable-dist-rustc-linux/build/src/libcore/result.rs:837
kablam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
kablam! 
boom! running `docker rm -f 8d4514314df7aa0324acbaff96e570654ec39997185c6022b029e56948c0fa27`
blam! 8d4514314df7aa0324acbaff96e570654ec39997185c6022b029e56948c0fa27
